docs/storage/storage-solution.md

```diff
@@ -2,25 +2,24 @@
 
 ## Concept
 
-Ability to run stateful workloads is a must. At the same time running stateful workloads is complex. Here the complexity is on another level considering the multi-/hybrid-cloud environment. Therefore, Claudie needs to be able to accommodate the stateful workloads, regardless of the underlying infrastructure providers.
+The ability to run stateful workloads is a must. At the same time, running stateful workloads is a complex task, even more so when considering the {multi,hybrid}-cloud environment. Therefore, Claudie needs to be able to accommodate the stateful workloads, regardless of the underlying infrastructure providers.
 
-Orchestrate storage on the kubernetes cluster nodes by creating one storage cluster across multiple providers. This storage cluster will have a series of `zones`, one for each cloud provider. Each `zone` should store its own persistent volume data.
+Claudie orchestrates storage on the kubernetes cluster nodes by creating one storage cluster across multiple providers. This storage cluster has a series of `zones`, one for each cloud provider. Each `zone` then stores its own persistent volume data.
 
 ## Longhorn
 
-Claudie created cluster comes with the longhorn deployment preinstalled and ready to be used. By default, only **worker** nodes are used to store data.
+A Claudie-created cluster comes with the `longhorn` deployment preinstalled and ready to be used. By default, only **worker** nodes are used to store data.
 
-Longhorn installed in the cluster is set up in a way, it provides one default `StorageClass` called `longhorn`, which if used, will create a volume which is replicated across random nodes in a cluster. 
+Longhorn installed in the cluster is set up in such a way that it provides one default `StorageClass` called `longhorn`, which - if used - creates a volume that is then replicated across random nodes in the cluster. 
 
-Other than default storage class, Claudie creates custom storage classes, which force persistent volumes to be created on a specific nodes based on their provider. In other words, you can use a specific provider to provision nodes for your storage needs, while using other provider for computing tasks.
+Besides the default storage class, Claudie can also create custom storage classes, which force persistent volumes to be created on a specific nodes based on the provider they have. In other words, you can use a specific provider to provision nodes for your storage needs, while using another provider for computing tasks.
 
 ## Example
 
-### [Input manifest](../input-manifest/example.yaml)
+To follow along, have a look at the reference [example input manifest file](../input-manifest/example.yaml).
 
-When Claudie will apply this input manifest, there will be a few storage classes installed. 
+When Claudie applies this input manifest, there are three storage classes installed:
```

Comment:
You specify `three`, but this might change in the future, and I'd be surprised if we remembered to update this word here.
It might be better to keep it vague.

Reply:
right, good point.

let's go with "the following"..